packages/components/stories/VirtualizedList.js

```diff
@@ -170,6 +172,24 @@ const collection = [
 	},
 ];
 
+let hugeDataSet = Array.from(new Array(NUMBER_ELEMENTS_HUGE_DATASET));
+
+hugeDataSet = hugeDataSet.map((item, index) => ({
```

Comment:
Yes ... that's the problem --> `You allocate an array of 100 elements just to use .map() instead of a for loop`